pkg/sidecar/config.go

```diff
@@ -32,16 +32,22 @@ type Config struct {
 	RenewSignalDeprecated              string `hcl:"renewSignal"`
 
 	// JWT configuration
-	JWTAudience       string `hcl:"jwt_audience"`
-	JWTSvidFilename   string `hcl:"jwt_svid_file_name"`
-	JWTBundleFilename string `hcl:"jwt_bundle_file_name"`
+	JwtSvids                  []JwtConfig `hcl:"jwt_svids"`
+	JWTAudienceDeprecated     string      `hcl:"jwt_audience"`
+	JWTSvidFilenameDeprecated string      `hcl:"jwt_svid_file_name"`
```

Comment:
Since we havent yet released this feature yet I dont think we need the deprecated flags, just delete them.

Reply:
👍 I was going to ask about that.